src/OrchardCore.Modules/OrchardCore.OpenId/OpenIdExtensions.cs

```diff
@@ -22,6 +22,12 @@ internal static string GetUserName(this ClaimsPrincipal principal)
            principal.FindFirst(ClaimTypes.Name)?.Value ??
            throw new InvalidOperationException("No suitable user name can be found in the principal.");
 
+    internal static string[] GetRoles(this ClaimsPrincipal principal)
+        => principal.FindAll(c => c.Type == Claims.Role || c.Type == ClaimTypes.Role)
+        .Select(x => x.ValueType)
```

Comment:
You probably wanted to use `Value` and not `ValueType` here 😃

Reply:
Fixed.